rpc_jobs/rpc_aio.yml

```diff
@@ -266,7 +266,7 @@
       env.ACTION = "{action}"
       env.SERIES = "{series}"
       timeout(time: 8, unit: 'HOURS'){{
-        node() {{
+        common.shared_slave() {{
```

Comment:
The common.groovy needs to be loaded before this in all the jobs. Not sure if we can clone repos outside of a node(), though.